database/import.go

```diff
@@ -61,7 +61,7 @@ func getHeadersFile(preparedDbFilePath string, log *zerolog.Logger) (*os.File, s
 		return nil, "", fmt.Errorf("file %s does not exist or is not readable", preparedDbFilePath)
 	}
 
-	const tmpHeadersFileName = "pulse-blockheaders.csv"
+	const tmpHeadersFileName = "bhs-blockheaders.csv"
```

Comment:
maybe just `blockheaders.tmp.csv`
or 
```suggestion
	tmpHeadersFileName := fmt.Sprintf("%d-blockheaders.csv", time.Now().Unix())
```

Reply:
done, I also updated .gitignore with wildcard schema

Reply:
good catch with this .gitignore 👍 🏆